### ISSUE TITLE
Live Region Not Announced

### ISSUE THREAD

**Issue by aardrian** [duetds/date-picker#28]
**Describe the bug**
I discovered a live region in the code:
```
<div class="duet-date__vhidden duet-date__instructions" aria-live="polite">You can use arrow keys to navigate dates</div>
```

However when I access the control I do not hear the live region announced.

**To Reproduce**
1. Go to the grid,
2. Interact with it (typing a date, opening the calendar, navigation it, choosing one, etc),
3. Observe the screen reader does not announce the content from the live region.

**Expected behavior**
I expect to hear the live region.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 85
 - JAWS 2020

**Additional context**
Using first example at https://duetds.github.io/date-picker/

**Comment by viljamis** [comment]
@aardrian Thank you for reporting this! I think we may have a fix for this coming soon-ish. We thought we already fixed this, but seems like it’s still an issue in some readers.

**Comment by WickyNilliams** [comment]
Was able to reproduce this in both JAWS and NVDA. VoiceOver does announce the region. Not sure why it is not getting announced in the others. I tried to fix by leaving the region blank until the picker is open (hoping that the screen readers would then see the text change, and then announce), but that didn't work either.

TBH live region kind of feels like the wrong choice here. It's not really live/dynamic, the content is static. But I'm unsure what would be more apporpriate

**Comment by WickyNilliams** [comment]
@aardrian I'm just going back through these issues. If you remember, we eventually settled on using a plain ol' table (rather than role="grid") for the calendar, so i'm thinking this help text is now redundant, since table navigation commands will be the standard for screen reader users rather than our custom keyboard shortcuts. What do you think? Should I drop it?

**Comment by aardrian** [comment]
@WickyNilliams I don't recall the instructions in place now, but if it is a table, announces as a table, and supports native table navigation in the screen reader, then there is no need to tell the screen reader user how to navigate it.

**Comment by WickyNilliams** [comment]
OK thanks, that was my intuition. 

It says "You can use arrow keys to navigate dates". This was true when we previously used the grid role since key presses get passed through to our code, but now it's a plain table the advice isn't just unnecessary, it's wrong! 

I'll remove it and resolve this bug that way :)
